pom.xml

```diff
@@ -44,6 +44,8 @@
   <properties>
     <hadoop.version>2.6.0-cdh5.5.0</hadoop.version>
     <spark.version>1.5.0-cdh5.5.0</spark.version>
+    <hadoop.scope>compile</hadoop.scope>
+    <spark.scope>compile</spark.scope>
```

Comment:
What is the point to check this in if the hadoop & spark scope override is removed?